src/components/button/button.module.scss

```
@@ -85,28 +86,31 @@
   --btn-padding-y: #{pxToRem(4)};
   --btn-padding-x: #{pxToRem(10)};
   --btn-font-size: #{pxToRem(13)};
+  min-height: #{$btn-height-sm};
```

Stylelint (17.4.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)